test/ussd_nurse_rapidpro.test.js

```diff
@@ -210,16 +210,35 @@ describe('app', function() {
                 });
 
                 describe('user enters msisdn', function(){
+
+                  //for msisdn that has not opted out
+                  it('should go to state_faccode', function(){
+                      return tester
+                      .setup.user.addr('27820001003')
+                      .inputs(
+                          {session_event: 'new'}
+                          ,'1' //chooses to start nurse connect
+                          ,'1'
+                          ,'1'
+                          ,'0820001003'  // state_msisdn
+                      )
+                      .check.interaction({
+                        state: 'state_faccode',
+                        reply: 'Please enter <your/their> 6-digit facility code:'
+                      })
+                      .run();
+                  });
+
                     //for msisdn that has opted out
                     it('should ask to opt in again', function(){
                         return tester
-                        .setup.user.addr('0000000000') //should be 27820001004 in fixtures
+                        .setup.user.addr('27820001004') //should be 27820001004 in fixtures
                         .inputs(
-                            {session_event: 'new'}  
+                            {session_event: 'new'}
                             ,'1' //chooses to start nurse connect
-                            ,'1'  // chooses to sign up
-                            ,'1' //chooses yes to subscribe
-                            ,'0000000000'  // state_msisdn
+                            ,'1'
```

Comment:
Please add comment on what this input is (similar to other test cases)

Reply:
thanks will do